packages/console/src/components/SubmitFormChangesActionBar/index.module.scss

```diff
@@ -0,0 +1,19 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  position: sticky;
```

Comment:
do we need the sticky layout here? not sure about the design.

Reply:
We do.
Refer to the design:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/10806653/202605137-60333a87-413f-43d4-a151-a7df30f8459d.png">

Reply:
That is "fixed"?

Reply:
`fixed` will cover the whole page width. We don't desire that.